cluster/images/etcd/Makefile

```diff
@@ -50,10 +50,20 @@ ifeq ($(ARCH),s390x)
 	BASEIMAGE?=s390x/busybox
 endif
 
+# Use GNU cp
+if LANG=C cp --help 2>&1 | grep -q GNU; then
+    CP="cp"
+elif which gcp &>/dev/null; then
+    CP="gcp"
+else
+    kube::log::error "Failed to find GNU cp as cp or gcp. If you are on Mac: brew install coreutils." >&2
+    return 1
+fi
+
 build:
 	# Copy the content in this dir to the temp dir,
 	# without copying the subdirectories.
-	find ./ -maxdepth 1 -type f | xargs cp -t $(TEMP_DIR)
+	find ./ -maxdepth 1 -type f | xargs ${CP} -t $(TEMP_DIR)
```

Comment:
I think the simpler fix would be to replace the right side of this with
`| xargs -I {} cp {} $(TEMP_DIR)`

Reply:
@ixdy Nice trick. Thanks.